macros/clone_schema.sql

```diff
@@ -1,11 +1,16 @@
-{% macro clone_schema(source_schema, destination_schema) %}
+{% macro clone_schema(source_schema, destination_schema, source_database=None, destination_database=None) %}
```

Comment:
Instead of setting the defaults to None and then doing conditions on line 5-6, you could simply do:

```
{% macro clone_schema(source_schema, destination_schema, source_database=target.database, destination_database=target.database) %}
```

Reply:
good call